src/services/ColorThemePlugin.js

```diff
@@ -54,5 +79,22 @@ export const ColorThemeMixin = {
       }
       return this.colorDefault
     },
+    isDefaultColorTheme () {
+      return this.$themesOptions.activeThemeName === this.$themesOptions.defaultThemeName
+    },
+  },
+}
+
+export const ColorThemeActionsMixin = {
```

Comment:
This code is never used.

Reply:
This code used in vuestic-admin components. Added comment